docs/reference/c-object-detection-2d-nanodet-jit-h.md

```diff
@@ -0,0 +1,47 @@
+## C_API: object_detection_2d_nanodet_jit.h
+
+
+The *object_detection_2d_nanodet_jit.h* header provides function definitions that can be used for accessing the OpenDR object detection 2d nanodet tool.
+
+### Struct *nanodet_model_t*
+```C
+
+struct nanodet_model {
+  ...
+};
+typedef struct nanodet_model nanodet_model_t;
+```
+The *nanodet_model_t* structure keeps all the necessary information that are required by the OpenDR object detection 2d nanodet tool (e.g., model weights, normalization information, etc.).
+
+
+### Function *load_nanodet_model()*
+```C
+void load_nanodet_model(char *modelPath, char *device, int height, int width, float scoreThreshold, nanodet_model_t *model);
```

Comment:
I am unsure if I fully understand the information provided in the wiki. It appears that all of the functions in the script are actually C++ functions that can be used in C. Given this, I have used C style naming conventions for the components (methods, and structures) that C scripts frequently use, while using C++ style for the variables so the scripts have C++ style in cpp files. Is it appropriate to mix the two styles in this way? Additionally, the structures in the project currently use C style, as was previously used in the opendr utils and face recognition modules. Could you provide guidance on the preferred naming convention in these cases, so that I can ensure consistency across the project and any modifications I make in the future?

Reply:
I cannot suggest anything specific as what you said is true, it is a bit confusing. @passalis any thoughts?

Reply:
I think the most important think here is to have a consistent API regardless of the actual conventions, i.e., not to mix different types of naming conventions for the same thing. Therefore, if there are no significant inconsistencies in the existing code base, I would propose to follow the style that it is already followed in the toolkit.